docs/user_guide/detectors/detector_computation.ipynb

```diff
@@ -24,213 +24,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
```

Comment:
This tutorial is out of date.

Reply:
It can be removed from `tqec` docs, I agree.